internal/db/queryBuilder.go

```diff
@@ -23,27 +23,38 @@ func newQueryBuilder(project int64, vrange VersionRange, objectQuery *pb.ObjectQ
 	}
 }
 
-func (qb *queryBuilder) possibleObjectsCTE() string {
-	return `
-		SELECT *
-		FROM dl.objects
-		WHERE project = __project__
-		  AND (
-		    -- live objects
-		    (
-		      start_version > __start_version__
-		      AND start_version <= __stop_version__
-		      AND (stop_version IS NULL OR stop_version > __stop_version__)
-		    )
-		    OR
-		    -- removed objects
-		    (
-		      start_version <= __stop_version__
-		      AND stop_version > __start_version__
-		      AND stop_version <= __stop_version__
-		    )
-		  )
-	`
+func (qb *queryBuilder) possibleObjectsCTE(includeRemovedObjects bool) string {
```

Comment:
nit: Could we name this `withRemovals` like we do with the method names? It was difficult to resolve that includeRemovedObjects == withRemovals for some reason 😆

Reply:
Yeah that's much better :+1: